### ISSUE TITLE
Relative Paths are not working on dotCMS 5.0

### ISSUE THREAD

**Issue by manuel-miranda** [dotCMS/core#15141]
Relative Paths are not working on dotCMS 5.0

## Expected Behavior
Relative paths like:
```html
<link media="all" rel="stylesheet" type="text/css" href="css/main.css">
```

## Current Behavior
Only absolute paths are working:
```html
<link media="all" rel="stylesheet" type="text/css" href="/css/main.css">
```
## Possible Solution
N/A

## Known Workarounds
Use absolute paths

## Steps to Reproduce (for bugs)
1. Create a page and add css with relative path

## Context
Slack thread: [Slack](https://dotcms.slack.com/archives/C0459G48Z/p1534515925000100)
Support Ticket reference: [#96399](https://dotcms.zendesk.com/agent/tickets/96399)

## Your Environment
* dotCMS version used: 5.0
* Browser Name and version: any
* Operating System and version: any
* Application Server and version:
* Java Brand and version:
* Database and version:
* Application module (if apply):

**Comment by dotcmsCodeReview** [comment]
This is not with sass, right?  Relative paths should in html should have
nothing to do with dotcms.  I am pretty sure they are working and something
else is happening.

**Comment by bryanboza** [comment]
Fixed, tested on release 5.0.2 and works as expected
